website/src/routes/api/(schemas)/string/index.mdx

```diff
@@ -2,6 +2,116 @@
 title: string
 ---
 
+import { Property } from '~/components';
+import { Link } from '@builder.io/qwik-city';
+
 # string
 
-> The content of this page is not yet ready. Until then just use the [source code](https://github.com/fabian-hiller/valibot/blob/main/library/src/schemas/string/string.ts).
+Schema that parses the input to string.
+
+```ts
+const schema = string(errorMessage, pipe);
+```
+
+## Parameter
+
+- `errorMessage` <Property type='string'/> = 'Invalid type'
+- `pipe` <Property type={{ type: 'custom', name: 'Pipe<string>' }}/> = []
+
+## Return
+
+- `schema` <Property type={{ type: 'custom', name: 'StringSchema' }}/>
+
+## Examples
+
+```ts
+const schema = string();
+parse(schema, 'string'); // returns 'string'
+parse(schema, 123); // throw ValiError 'Invalid Type'
+
+const schemaWithCustomErrMsg = string('custom error message');
+parse(schemaWithCustomErrMsg, 123); // throw ValiError 'custom error message'
+
+string([maxLength(5)]);
+string([minLength(5)]);
+string([length(5)]);
+string([email()]);
+string([url()]);
+string([emoji()]);
+string([uuid()]);
+string([cuid2()]);
+string([ulid()]);
+string([regex(regex)]);
+string([includes(string)]);
+string([startsWith(string)]);
+string([endsWith(string)]);
+string([datetime()]);
+string([ip()]);
+```
+
+## Methods
```

Comment:
![badge](https://img.shields.io/badge/review-memo-blue.svg)
Methods which can be used with string()

Reply:
Yes, thats great! Good idea!

Reply:
Here I would only add the relevant methods. I would remove `coerceAsync`, `fallbackAsync`, `getDefault`, `parseAsync`, `safeParseAsync` and `unwrap`.

Reply:
Please also remove `transformAsync`

Reply:
Fixed!

https://github.com/fabian-hiller/valibot/pull/220/commits/ad5a6d42cc6280673cb9e388eee1baec7eda5793
https://github.com/fabian-hiller/valibot/pull/220/commits/95be95dc84efb7970a2b459717db7a1d16f64d7a